icons/comma.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>comma</title><path d="M6.7084 24c0-.55386-.04365-1.02045.02013-1.47025.02686-.19134.25176-.3961.43638-.51023.88954-.56393 1.85628-1.01374 2.6854-1.65823 2.6854-2.09125 4.27649-4.84714 4.36376-8.42543.0235-1.01374-.3491-1.26885-1.20842-.87276-2.47057 1.1413-5.12575.46995-6.54566-1.65151a6.0455 6.0455 0 0 1 .52366-7.4184c2.36986-2.5847 6.26368-2.6619 8.94908-.1947 1.61124 1.47697 2.37322 3.38695 2.56455 5.55541.63778 7.27407-3.02107 13.26586-9.6842 15.9009-.65793.25847-1.33264.4733-2.10469.7452"/></svg>
```

Comment:
I'm not seeing any differences after saving, optimizing, and reopening with the following:

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" ><title>comma</title><path d="M6.72682 24c0-.55531-.0434-1.02051.02045-1.46891.0272-.1912.25037-.39595.43237-.51143.88731-.56315 1.85122-1.01275 2.67734-1.65857 2.67584-2.09206 4.26201-4.84612 4.3487-8.42366.02452-1.01583-.34891-1.2696-1.20211-.87389-2.4628 1.1424-5.11119.47263-6.5246-1.65034-1.54137-2.3155-1.32431-5.3984.52253-7.4211 2.359-2.58344 6.24053-2.66074 8.91722-.19346 1.60337 1.47794 2.3652 3.38627 2.5552 5.5569.63691 7.27188-3.01046 13.2657-9.64881 15.89874-.657.26045-1.3307.4744-2.09828.74571"/></svg>
```